addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.udf_ui",
+    "name": "User Defined Functions Formula Editor",
+    "description": "A Seeq add-on tool for managing user-defined formula functions.",
```

Comment:
[nit] `add-on` -> `Add-on` (consider also capitalizing Tool)

Reply:
Fixed in the next push!